.github/workflows/CI.yml

```diff
@@ -136,14 +136,14 @@ jobs:
       needs.setup.outputs.workflows != 'false'
     uses: ./.github/workflows/validate_filenames.yml
 
-  validate_generated_files:
-    name: Validate
-    needs: [setup]
-    if: >-
-      needs.setup.outputs.frontend != 'false' ||
-      needs.setup.outputs.config != 'false' ||
-      needs.setup.outputs.workflows != 'false'
-    uses: ./.github/workflows/validate_generated_files.yml
+  # validate_generated_files:
+  #   name: Validate
+  #   needs: [setup]
+  #   if: >-
+  #     needs.setup.outputs.frontend != 'false' ||
+  #     needs.setup.outputs.config != 'false' ||
+  #     needs.setup.outputs.workflows != 'false'
+  #   uses: ./.github/workflows/validate_generated_files.yml
```

Comment:
This is no longer needed since all the files should be generated now, and it has nothing to verify but let's keep it around for now but commented out.